Crash when resume

```claude --resume a93f1802-cc9e-4470-a7f5-1d7dcf5044e5 "$(cat /var/folders/cg/b87mbnkn0vqdtchfvdwq3_f00000gp/T/ccrider-prompt-214199101.txt)"
panic: send on closed channel

goroutine 1 [running]:
github.com/neilberkman/ccrider/internal/core/session.(*Spinner).Stop(...)
        /Users/runner/work/ccrider/ccrider/internal/core/session/spinner.go:50
github.com/neilberkman/ccrider/internal/interface/cli.execClaude({0x140001da660, 0x24}, {0x14000034150, 0x18}, {0x14000034180, 0x18}, {0x140009a78c0, 0x13}, {0x1400019e200, 0x33}, ...)
        /Users/runner/work/ccrider/ccrider/internal/interface/cli/tui.go:171 +0x9d8
github.com/neilberkman/ccrider/internal/interface/cli.runTUI(0x14000043ae0?, {0x0?, 0x0?, 0x0?})
        /Users/runner/work/ccrider/ccrider/internal/interface/cli/tui.go:55 +0x2d4
github.com/neilberkman/ccrider/internal/interface/cli.init.func1(0x14000219500?, {0x1062177e0?, 0x7?, 0x1057c0a47?})
        /Users/runner/work/ccrider/ccrider/internal/interface/cli/root.go:39 +0x30
github.com/spf13/cobra.(*Command).execute(0x1061e4640, {0x14000194020, 0x0, 0x0})
        /Users/runner/go/pkg/mod/github.com/spf13/cobra@v1.10.1/command.go:1015 +0x844
github.com/spf13/cobra.(*Command).ExecuteC(0x1061e4640)
        /Users/runner/go/pkg/mod/github.com/spf13/cobra@v1.10.1/command.go:1148 +0x384
github.com/spf13/cobra.(*Command).Execute(...)
        /Users/runner/go/pkg/mod/github.com/spf13/cobra@v1.10.1/command.go:1071
github.com/neilberkman/ccrider/internal/interface/cli.Execute()
        /Users/runner/work/ccrider/ccrider/internal/interface/cli/root.go:24 +0x24
main.main()
        /Users/runner/work/ccrider/ccrider/cmd/ccrider/main.go:16 +0x130
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,5 +1,11 @@
 # Changelog
 
+## [0.9.9] - 2026-01-12
+
+### Fixed
+
+- **Spinner panic on resume** - fixed "send on closed channel" crash when resuming sessions. The spinner's Stop() method is now idempotent, safe to call multiple times (fixes #2).
+
 ## [0.9.8] - 2026-01-12
 
 ### Added

--- a/internal/core/session/spinner.go
+++ b/internal/core/session/spinner.go
@@ -4,14 +4,16 @@ import (
 	"fmt"
 	"io"
 	"os"
+	"sync"
 	"time"
 )
 
 // Spinner shows a simple spinning animation while waiting
 type Spinner struct {
-	writer  io.Writer
-	message string
-	stop    chan bool
+	writer   io.Writer
+	message  string
+	stop     chan struct{}
+	stopOnce sync.Once
 }
 
 // NewSpinner creates a new spinner with a message
@@ -19,7 +21,7 @@ func NewSpinner(message string) *Spinner {
 	return &Spinner{
 		writer:  os.Stderr,
 		message: message,
-		stop:    make(chan bool),
+		stop:    make(chan struct{}),
 	}
 }
 
@@ -45,8 +47,9 @@ func (s *Spinner) Start() {
 	}()
 }
 
-// Stop stops the spinner and clears the line
+// Stop stops the spinner and clears the line. Safe to call multiple times.
 func (s *Spinner) Stop() {
-	s.stop <- true
-	close(s.stop)
+	s.stopOnce.Do(func() {
+		close(s.stop)
+	})
 }

